package.json

```diff
@@ -130,6 +130,7 @@
   },
   "dependencies": {
     "@mantine/core": "7.1.3",
+    "@mantine/form": "^7.1.5",
```

Comment:
This version is higher then other `mantine` packages and not static like other. I think in package files need to use the same version notations. And please use same versions for `mantine` packages because each `mantine` packages releases with same version